test/evergreen/compatibility_test_for_mongodb_releases.sh

```diff
@@ -85,16 +99,46 @@ verify_release()
         echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
         echo "Release \"$1\" verifying \"$2\""
         echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
-        
+
         cd "$1"
         for am in $3; do
-            dir="$2/test/format/RUNDIR.$am"
             echo "$1/wt verifying $2 access method $am..."
-
+            dir="$2/test/format/RUNDIR.$am"
             WIREDTIGER_CONFIG="$EXT" ./wt -h "../$dir" verify table:wt
         done
 }
 
+#############################################################
+# upgrade_downgrade:
+#       arg1: release #1
+#       arg2: release #2
```

Comment:
Does `mongo release branch` feel more readable compared to `release`?

Reply:
Hmmmm, I think you're right, the whole script should be talking about `branches`, not `releases`.

Pushed in 74f019c85.